.github/workflows/main.yml

```diff
@@ -21,11 +21,14 @@ on:
 
 env:
   YARN_ENABLE_HARDENED_MODE: 0
+  NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
 
 jobs:
   # Check the integrity of yarn lock
   yarn_lock_check:
     runs-on: ubuntu-latest
+    env:
+      NX_SKIP_NX_CACHE: true
```

Comment:
Not related to your change but don't we want the Nx cache for this step?

Reply:
It is requires by `nx-cloud` runner when run in CI.
cf: https://github.com/AmadeusITGroup/otter/actions/runs/11702511447/job/32590842412